sql_generators/glean_usage/templates/cross_channel_events_unnested.view.sql

```diff
@@ -65,6 +65,7 @@ SELECT
   event.category AS event_category,
   event.name AS event_name,
   event.extra AS event_extra,
+  "{{ dataset }}" AS normalized_app_id
```

Comment:
Won't the underlying `events` views already have a `normalized_app_id` column now?

Reply:
It's using the `event` views from the specific channels (e.g. `org_mozilla_fennec_aurora`) which don't have this information (it's only in the unioned app views like `fenix`)

Reply:
Although we could update the view generation logic at some point to use the new unioned app ping views instead of the channel-specific views.

Reply:
Ah, I see, those are generated by `stable_views` rather than `glean_usage`.